docs/ref/contrib/admin/index.txt

```diff
@@ -965,6 +965,22 @@ subclass::
     editing, or deleting an object. You can have filters cleared by setting
     this attribute to ``False``.
 
+.. attribute:: ModelAdmin.show_facets
+
+    .. versionadded:: 5.0
+
+    Set ``show_facets`` to ``True`` to have all filter options appended with
+    the count of results that match that filter option in brackets. These
```

Comment:
```suggestion
    the count of results that match that filter option. These
```

It took me a few attempts to understand the 'in brackets' part. Maybe we can drop that part? 🤷‍♂️

Reply:
Yeah no problem :+1: